tools/pipelines/templates/include-install-pnpm.yml

```
@@ -48,5 +48,36 @@ steps:
       echo "Using node $(node --version)"
       sudo corepack enable
       echo "Using pnpm $(pnpm -v)"
+      # All below settings are user-level. We set the user-level .npmrc to be located in a well-known temp location
+      # that will persist for the duration of this job.
+      echo "##vso[task.setvariable variable=NPM_CONFIG_USERCONFIG]$(Agent.TempDirectory)/.npmrc"
```

Since we provide the value through `env:`, would this work to avoid repeating the path?

```suggestion
      echo "##vso[task.setvariable variable=NPM_CONFIG_USERCONFIG]$NPM_CONFIG_USERCONFIG"
```